src/graph/entries.mo

```diff
@@ -0,0 +1,45 @@
+import Array "mo:stdlib/array.mo";
+import Blob "mo:stdlib/blob.mo";
+import HashMap "mo:stdlib/hashMap.mo";
+import Hash "mo:stdlib/hash.mo";
+import Iter "mo:stdlib/iter.mo";
+import List "mo:stdlib/list.mo";
+import Nat "mo:stdlib/nat.mo";
+import Option "mo:stdlib/option.mo";
+
+import Types "./types.mo";
+
+module {
+  type Entry = Types.Entry;
+  type EntryId = Types.EntryId;
+
+  public class Entries () {
+
+    func natEq (x : Nat32, y : Nat32) : Bool { x == y };
```

Comment:
Suggestion: spaces between function name and parameter list is not typical style in C-like languages. I would avoid it.